server/app.js

```diff
@@ -46,6 +46,7 @@ app.use((req, res, next) => {
 app.use("/auth", authRouter);
 app.use("/users", userRouter);
 app.use("/profile", profileRouter);
+app.use("/conversation",conversationRouter);
```

Comment:
```suggestion
app.use("/conversations",conversationRouter);
```
Let's update the route name as per REST guidelines

Reply:
Done the changes.